src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/UcWriteFinalDecisionIt.java

```diff
@@ -224,17 +228,17 @@ public void callToMidEventPreviewFinalDecisionCallback_willPreviewTheDocumentFor
         assertEquals("2018-10-10",payload.getStartDate());
         assertEquals("2018-11-10",payload.getEndDate());
         assertEquals(false, payload.isIndefinite());
-        assertEquals(true, payload.isEsaIsEntited());
-        assertEquals("lower rate", payload.getEsaAwardRate());
-        Assert.assertNotNull(payload.getEsaSchedule2Descriptors());
-        assertEquals(1, payload.getEsaSchedule2Descriptors().size());
-        assertEquals(15, payload.getEsaSchedule2Descriptors().get(0).getActivityAnswerPoints());
-        assertEquals("a", payload.getEsaSchedule2Descriptors().get(0).getActivityAnswerLetter());
-        assertEquals("Cannot, unaided by another person, either: (i) mobilise more than 50 metres on level ground without stopping in order to avoid significant discomfort or exhaustion; or (ii) repeatedly mobilise 50 metres within a reasonable timescale because of significant discomfort or exhaustion.", payload.getEsaSchedule2Descriptors().get(0).getActivityAnswerValue());
-        assertEquals("1. Mobilising unaided by another person with or without a walking stick, manual wheelchair or other aid if such aid is normally or could reasonably be worn or used.", payload.getEsaSchedule2Descriptors().get(0).getActivityQuestionValue());
-        assertEquals("1", payload.getEsaSchedule2Descriptors().get(0).getActivityQuestionNumber());
-        Assert.assertNotNull(payload.getEsaNumberOfPoints());
-        assertEquals(15, payload.getEsaNumberOfPoints().intValue());
+        assertEquals(true, payload.isUcIsEntited());
```

Comment:
should this be entitled?

Reply:
No, it shouldn't - good spot Chris - thanks!   Have now changed this test